Snippets/AccessManager/01-access-manager.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  01-access-manager.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+
+// snippet.import
+import PubNubSDK
+
+// snippet.end
+
+// snippet.pubnub
+// Initializes a PubNub object with the configuration
```

Comment:
Can't this be extracted to base class and reuse across different files?

Reply:
That would be nice. I will run it by the docs team, since this would require `EmbeddedCode` component to take snippets from more than one URL